Lib/test/test_sqlite.py

```diff
@@ -18,7 +18,7 @@ def load_tests(*args):
                                userfunctions.suite(),
                                factory.suite(), transactions.suite(),
                                hooks.suite(), regression.suite(),
-                               dump.suite()])
+                               dump.suite(), backup.suite()])
```

Comment:
Nitpick: Add this to its own line so we won't clutter `git blame` next time we add something.